flashcards/backend/decks/tests/test_validators.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+from django.core.exceptions import ValidationError
+from django.core.files.uploadedfile import SimpleUploadedFile
+
+from config import settings
+from decks.models import Deck
+from decks.validators import validate_file_type
+
+
+def test_negative_price_raises_value_error(user, db):
+    deck = Deck(
+        name="test",
+        author_id=user.id,
+        description="test",
+        price=-1,
+    )
+    with pytest.raises(ValidationError) as excinfo:
+        deck.full_clean()
+    assert "Ensure this value is greater than or equal to 0.00" in str(excinfo.value)
+
+
+def test_valid_image(image):
+    assert not validate_file_type(image)
+
+
+def test_not_valid_image_with_correct_extension():
```

Comment:
I suggest generating new Image with Pillow package as test image it will reduce the amount of test data in our repository.

https://pillow.readthedocs.io/en/stable/reference/Image.html#PIL.Image.new

Reply:
i suggest new task for this for more clever persons, it is easy to create new image but how to change content of this image later ti run validator tests? dozen lines of code vs one image

Reply:
done